src/locales/strings/enUS.json

```diff
@@ -160,7 +160,7 @@
   "fragment_transaction_list_tx_failed": "Failed",
   "fragment_transaction_list_tx_synchronizing": "Syncing...",
   "fragment_transaction_list_confirmation_progress": "%s of %s Confirmations",
-  "fragment_transaction_list_unconfirmed_rbf": "Awaiting confirmation, may be cancelled",
+  "fragment_transaction_list_unconfirmed_rbf": "Awaiting confirmation; may be cancelled",
```

Comment:
Also update other applicable languages since this string is already partially localized. 

Maybe also change the string key too, because I don't know if crowdin would cache the original string.